iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergTimeTravel.java

```diff
@@ -36,15 +34,19 @@ public class TestHiveIcebergTimeTravel extends HiveIcebergStorageHandlerWithEngi
 
   @Test
   public void testSelectAsOfTimestamp() throws IOException, InterruptedException {
-    Table table = prepareTableWithVersions(2);
+    Table table = testTables.createTableWithVersions(shell, "customers",
+        HiveIcebergStorageHandlerTestUtils.CUSTOMER_SCHEMA,
+        fileFormat, HiveIcebergStorageHandlerTestUtils.CUSTOMER_RECORDS, 2);
 
     List<Object[]> rows = shell.executeStatement(
-        "SELECT * FROM customers FOR SYSTEM_TIME AS OF '" + timestampAfterSnapshot(table, 0) + "'");
+        "SELECT * FROM customers FOR SYSTEM_TIME AS OF '" +
+            HiveIcebergTestUtils.timestampAfterSnapshot(table, 0) + "'");
```

Comment:
nit: maybe static import the method, and then there is less code and code change?